packages/fxa-payments-server/README.md

```diff
@@ -72,6 +72,16 @@ Note that prior to testing you may need to create a build of the React App. You
 
 Refer to Jest's [CLI documentation](https://jestjs.io/docs/en/cli) for more advanced test configuration.
 
+### Debugging Tests
+
+A launch config for Visual Studio Code is provided to help debug tests. In order to use the launch config, load the `fxa-payments-server` workspace in Visual Studio Code, then add a `debugger` statement in the body of the test you plan to debug. In the Visual Studio Code sidebar navigate to the Run and Debug tab, and select the "Debug FxA-Payments Tests." Click the run button and the tests will begin to run and a debugger will be attached to the process. You can now add further breakpoints to the test source code by clicking near the line number.
+
+#### Current Debugging Limitations
```

Comment:
lets add some wording around which tests can be debugged. things found in server dir vs src

Reply:
done